src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1964,20 +1964,11 @@ public int getExperiencePointsNeededForNextLevel()
 		throw new UnimplementedOperationException();
 	}
 
-
-	@Nullable
 	@Override
-	public Location getBedSpawnLocation()
-	{
-		return bedSpawnLocation;
-	}
+	public @Nullable Location getRespawnLocation() { return this.respawnLocation; }
```

Comment:
This here is what I meant reformat the code. Here's an example
```suggestion
	public @Nullable Location getRespawnLocation()
	{ 
		return this.respawnLocation; 
	}
```

Reply:
Ok, this should have been reformatted now.